job_conf.xml

```diff
@@ -100,6 +100,9 @@
 	<tool id="clinical_pathology_images" destination="condor_1C_100M_central_gateway"/>
 	<tool id="icgc_genomes" destination="condor_1C_100M_central_gateway"/>
 	<tool id="ga4gh" destination="condor_1C_100M_central"/>
+        <tool id="mutsig_viz" destination="condor_1C_100M"/>
+        <tool id="hpt_viz" destination="condor_1C_100M"/>
+        <tool id="multi_rnaseq" destination="condor_1C_100M"/>
```

Comment:
If you need these tools to run on the central function, use the destination "condor_1C_100M_central" (see the previous entries in the job_conf.xml)
If you need these tools to run on the gateway node of the central function, use the destination "condor_1C_100M_central_gateway"